examples/dgfip_c/ml_primitif/promote_tests.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+year=${1:-2020}
```

Comment:
I guess you could use something like 2000 for lower bound :p

Reply:
It's just the default value, we don't even have M code for 2000.

Reply:
Alright, I must still be sleeping, I thought it was a range. Seems I've just learnt how to specify default value for arguments in bash.